roles/pulp_common/tasks/configure.yml

```diff
@@ -17,4 +17,29 @@
         mode: 0640
         force: yes
 
+    - name: Create wrapper to call pulpcore-manager
+      template:
+        src: pulpcore-manager.j2
+        dest: "/usr/local/bin/pulpcore-manager"
+        owner: root
+        group: "{{ pulp_group }}"
+        mode: "0750"
+
+    - name: Create directory for sudoer snippets
+      file:
+        path: "/etc/sudoers.d"
+        state: directory
+        owner: root
+        group: root
+        mode: 0755
```

Comment:
This changes permissions on the Red Hat family from 750 to 755. And organizations often care about permissions on this directory.

I think the more appropriate solution is to install the `sudo` package as part of pulp_preq_packages.

The alternative is to define the permissions with a variable per-distro.

Test results:
```
debian:10: 755 permissions. Not in default container until you install `sudo`.
fedora:32: 750 permissions. Present in default container
fedora:33: 750 permissions. Present in default container
centos:7: 750 permissions. Not in default container until you install `sudo`.
```

Reply:
we need sudo anyway for this to work...
So should i install it?